ethereum/core/src/main/java/tech/pegasys/artemis/core/EpochProcessorUtil.java

```diff
@@ -355,9 +354,9 @@ private static UnsignedLong get_base_reward(BeaconState state, int index) {
           UnsignedLong increment = EFFECTIVE_BALANCE_INCREMENT;
           final UnsignedLong reward_numerator =
               index_base_reward.getValue().times(attesting_balance.dividedBy(increment));
-          rewards.set(index, reward_numerator.dividedBy(total_balance.dividedBy(increment)));
```

Comment:
Apparently the spec had this same bug: https://github.com/ethereum/eth2.0-specs/pull/1680/files